svelte/bootstrap/src/components/tree/Tree.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+	import {Slot} from '@agnos-ui/svelte-headless/slot';
+	import {callWidgetFactory} from '../../config';
+	import type {TreeContext, TreeProps, TreeSlotItemContext} from './tree.gen';
+	import {createTree} from './tree.gen';
+	import TreeDefaultItem from './TreeDefaultItem.svelte';
+	import TreeDefaultRoot from './TreeDefaultRoot.svelte';
+	import TreeDefaultStructure from './TreeDefaultStructure.svelte';
+	import TreeDefaultToggle from './TreeDefaultToggle.svelte';
+
+	let {...props}: Partial<TreeProps> = $props();
+
+	const widget = callWidgetFactory({
+		factory: createTree,
+		widgetName: 'tree',
+		get props() {
```

Comment:
As you do not have a bindable prop, you do not need a getter. But `enablePatchChanged` is missing. So the end result should be:
```typescript
let props: Partial<TreeProps> = $props();

const widget = callWidgetFactory({
  factory: createTree,
  widgetName: 'tree',
  props,
  enablePatchChanged: true,
  defaultConfig: {structure, root, item, toggle},
});
```